.settings/org.eclipse.core.resources.prefs

```diff
@@ -1,2 +1,7 @@
 eclipse.preferences.version=1
+encoding//src/main/java=UTF-8
+encoding//src/main/resources=UTF-8
+encoding//src/test/java=UTF-8
+encoding//src/test/resources=UTF-8
 encoding/<project>=UTF-8
+encoding/rest-api-spec=UTF-8
```

Comment:
If we specify the encoding at the project level, shouldn't all source folders inherit it?